proto/paladin.proto

```diff
@@ -47,13 +47,29 @@ message PaladinTalents {
 
 message RetributionPaladin {
     message Rotation {
-		bool consecration = 1;
+		enum ConsecrateRank {
+			None = 0;
+			Rank1 = 1;
+			Rank4 = 2;
+			Rank6 = 3;
+		}
+		ConsecrateRank consecrateRank = 1;
+		bool exorcism = 2;
     }
     Rotation rotation = 1;
 
     PaladinTalents talents = 2;
     
     message Options {
+		enum Judgement {
+			None = 0;
+			Wisdom = 1;
+			Crusader = 2;
+		};
+		Judgement judgement = 1;
+		int32 csDelay = 2;
+		int32 hasteLeeway = 3;
+		int32 damageTaken = 4;
```

Comment:
named_like_this

I know it's verbose but could you spell out crusader_strike_delay? It's important that the code be readable by non-paladins.

Is damageTaken supposed to be damage_taken_per_second? If so you might also want to make this a double instead of an int.